react_juce/core/TextView.h

```diff
@@ -38,21 +38,27 @@ namespace blueprint
         TextView() = default;
 
         //==============================================================================
-        /** Assembles a Font from the current node properties. */
-        juce::Font getFont()
+        /** Assembles a Font from properties. */
+        static juce::Font getFont(const juce::NamedValueSet &_props)
         {
-            float fontHeight = props.getWithDefault(fontSizeProp, 12.0f);
-            int textStyleFlags = props.getWithDefault(fontStyleProp, 0);
+            float fontHeight = _props.getWithDefault(fontSizeProp, 12.0f);
```

Comment:
Super picky but `fontHeight` and `textStyleFlags` could be const

Reply:
I fixed it in c724958.